api/client.go

```diff
@@ -50,6 +51,7 @@ type Client struct {
 	Compliance      *ComplianceService
 	Integrations    *IntegrationsService
 	Vulnerabilities *VulnerabilitiesService
+	V2              *V2Endpoints
```

Comment:
Are all v2 endpoints going to live in the V2Endpoints service?

Reply:
@dmurray-lacework do you see a different pattern? They are all based on Schemas and there are not sections or grouping. 🤔 

They are all `/v2/{endpoint}` vs V1 that are `/v1/external/suppressions/{endpoint}` and `/v1/external/vulnerability/{endpoint}`

Let us sync up today about this since it is a very important design pattern.